pages/launcher.vue

```diff
@@ -0,0 +1,1852 @@
+<script setup>
+import { TrashIcon, SearchIcon, BoxIcon, SendIcon, EditIcon, DownloadIcon } from 'omorphia'
+import Avatar from '~/components/ui/Avatar.vue'
+import homepageProjects from '~/generated/homepage.json'
+import LogoAnimated from '~/components/brand/LogoAnimated.vue'
+import Badge from '~/components/ui/Badge.vue'
+import PrismIcon from '~/assets/images/external/prism.svg'
+import ATLauncher from '~/assets/images/external/atlauncher.svg'
+import Checkbox from '~/components/ui/Checkbox.vue'
+
+const val = Math.ceil(homepageProjects.length / 5)
+const os = ref(null)
+
+const rows = shallowRef([
+  homepageProjects.slice(0, val),
+  homepageProjects.slice(val, val * 2),
+  homepageProjects.slice(val * 2, val * 3),
+  homepageProjects.slice(val * 3, val * 4),
+  homepageProjects.slice(val * 4, val * 5),
+])
+
+onMounted(() => {
+  os.value = navigator?.platform.toString()
+})
+</script>
+
+<template>
+  <div>
+    <div class="landing-hero">
+      <h1 class="main-header">
+        Download Modrinth <br />
+        app
+        {{
+          os ? `for ${os?.includes('Mac') ? 'Mac' : os?.contains('Win') ? 'Windows' : 'Linux'}` : ''
```

Comment:
```suggestion
          os ? `for ${os?.includes('Mac') ? 'Mac' : os?.includes('Win') ? 'Windows' : 'Linux'}` : ''
```

`String.prototype.contains` is not a function.